### ISSUE TITLE
When I enable the NNPACK option, the network 's output is incorrect.

### ISSUE THREAD

**Issue by yjcn** [shizukachan/darknet-nnpack#3]
When I enable the NNPACK option, the network 's output is incorrect. And I disable NNPACK, the result is right.
![image](https://user-images.githubusercontent.com/8858803/35395118-703ca1f6-0224-11e8-833f-b796a70d8598.png)
![image](https://user-images.githubusercontent.com/8858803/35395124-7415c4ce-0224-11e8-8ff4-7dddf0eedada.png)

**Comment by shizukachan** [comment]
I think you need to make sure you're calling nnp_initialize if you are using different functionality in the code.

For an example of the necessary changes, look at https://github.com/shizukachan/darknet-nnpack/commit/b9402ac82c6935ea5040c37ad321e868399a05f7

**Comment by yjcn** [comment]
@shizukachan  Yes. Thank you for your reminding. I forget add "nnp_initialize();" and "net->threadpool = pthreadpool_create(4); ". Now I use digitalbrain79 's  [darknet-nnpack](https://github.com/digitalbrain79/darknet-nnpack) and the result is correct, but the speed isn't faster(don't use nnpack 46ms per image; use nnpack 50ms per image ).  And I try to use your darknet-nnpack, the result is still incorrect. Besides "nnp_initialize();",should I complete other initialization?
